src/main/java/emissary/core/BaseDataObject.java

```diff
@@ -369,6 +371,23 @@ public SeekableByteChannelFactory getChannelFactory() {
         }
     }
 
+    /**
```

Comment:
Don't duplicate the javadoc comment from the interface method. It will inherit by default, and you can explicitly specify inheritence by using the following:
```java
/**
 * {@inheritDoc}
 */
```    
Duplicating just increases the likelihood of unintended future drift.